yt/frontends/fits/data_structures.py

```diff
@@ -69,11 +67,13 @@
 sky_prefixes = list(sky_prefixes)
 spec_prefixes = list(spec_names.keys())
 
-field_from_unit = {"Jy":"intensity",
-                   "K":"temperature"}
+field_from_unit = {"Jy": "intensity",
```

Comment:
Do we ever expect these to be imported outside of this file?

Reply:
Nope, I'll fix that.